src/registries/aave-v2.ts

```diff
@@ -6,11 +6,13 @@ import { ILendingPoolAddressesProviderV2, ABI as ILendingPoolAddressesProviderV2
 import { Address } from "../pair"
 import { initPairsAndFilterByWhitelist } from "../utils"
 import { PairATokenV2 } from "../pairs/atoken-v2"
+import { Registry } from "../registry"
 
-export class RegistryAaveV2 {
+export class RegistryAaveV2 extends Registry {
 	private provider: ILendingPoolAddressesProviderV2
 
-	constructor(private kit: ContractKit, lendingPoolAddrProviderAddr: string) {
+	constructor(private kit: ContractKit, lendingPoolAddrProviderAddr: string, name?: string) {
```

Comment:
if we are adding names it should just be a required arg at this point. It is ok if it isn't backwards compatible, it is all typed so it will just throw compilation errors. 

So I think `name` should just be first argument in all classes and a required one if it is getting added.

Reply:
Okay updating to make name required arg.